.github/workflows/docs.yml

```diff
@@ -47,13 +47,8 @@ jobs:
           rm -rf build/docs/.doctrees
           rm build/docs/.buildinfo
 
-      - name: Adding in .nojekyll and CNAME
-        run: |
-          touch build/docs/.nojekyll
-          echo "symforce.org" > build/docs/CNAME
-
       - name: Deploy docs
-        uses: cpina/github-action-push-to-another-repository@v1.5
+        uses: symforce-org/github-action-push-to-another-repository@main
```

Comment:
I think it's worth pinning to the commit 10a3e9ec247be83fedd82f7bb7c3ed1f7218528e here (as [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsuses)), especially without branch protection on the action repo

Reply:
Good idea.

Reply:
Done. Am retesting the workflow here: https://github.com/bradley-solliday-skydio/symforce/runs/7460729320?check_suite_focus=true